pkg/sql/colexec/agg/anyvalue/types.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package anyvalue
+
+import "github.com/matrixorigin/matrixone/pkg/container/types"
+
+type Anyvalue[T any] struct {
+	// the len of NotSet should as same as agg's size
```

Comment:
What do you mean by "agg's size"?    Number of groups?   Number of elements in one group?   Or the number of bytes of one element in one group?

Reply:
> What do you mean by "agg's size"? Number of groups? Number of elements in one group? Or the number of bytes of one element in one group?

number of groups

Reply:
Sorry I will remove this annotation. I add it before implementing Grow() function. 
Its size will grow with group size when calling Grow() function.